AlphaWallet/Core/Types/MagicLinkUrl.swift

```diff
@@ -0,0 +1,167 @@
+//
+//  MagicLinkUrl.swift
+//  AlphaWallet
+//
+//  Created by Vladyslav Shepitko on 14.01.2022.
+//
+
+import Foundation
+
+enum MagicLinkUrl {
```

Comment:
Would you help rename this type? "MagicLink" refers to `.claimOrderMagicLink`. The link looks something like: https://aw.app/AQABhqBiIHUYZvCMpokgF6Rdpvt5Ko6Ub8vasfaj2GVa-oxrasfamYtJaLI6ofGQABVKXatbBXzDhozu2xuw3BFG4oc7fzuovT362HbjQDpQ0Ac-w-VIHhMYzVOB5fXNZwmwzhw= (this does't work)

Reply:
Maybe `DeepLink`?

Reply:
@oa-s 

Just to clarify: 
- MagicLinkUrl -> DeepLinkUrl
- .claimOrderMagicLink -> .magicLink?

Reply:
> MagicLinkUrl -> DeepLinkUrl

Yes. Or just `DeepLink`

> .claimOrderMagicLink -> .magicLink?

Yes.

Reply:
> Would you help rename this type? "MagicLink" refers to `.claimOrderMagicLink`. The link looks something like: https://aw.app/AQABhqBiIHUYZvCMpokgF6Rdpvt5Ko6Ub8vasfaj2GVa-oxrasfamYtJaLI6ofGQABVKXatbBXzDhozu2xuw3BFG4oc7fzuovT362HbjQDpQ0Ac-w-VIHhMYzVOB5fXNZwmwzhw= (this does't work)

here we got ecrecover error, but signedOrder was detected. so do we need to make  ecrecover for detecting address? any way this call will be performed next time

Reply:
thanks, link works for me ;)